migrations/sqls/20201028204537-followups-up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS followUps (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  clientId INT NOT NULL,
+  title VARCHAR(255) NOT NULL,
+  description MEDIUMTEXT,
+  dateOfContact DATETIME,
+  appointmentDate DATETIME,
+  addedBy INT NOT NULL,
+  updatedBy INT NOT NULL,
+  FOREIGN KEY (clientId) REFERENCES clients(id),
+  FOREIGN KEY (addedBy) REFERENCES users(id),
+  FOREIGN KEY (updatedBy) REFERENCES users(id),
+)
```

Comment:
I thought MySQL required semicolons. Is this working without them? I would add them in, to be safe.